specification/web/resource-manager/Microsoft.Web/stable/2020-09-01/ResourceProvider.json

```diff
@@ -0,0 +1,1398 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-09-01",
+    "title": " API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.Web/generateGithubAccessTokenForAppserviceCLI": {
+      "post": {
+        "summary": "Exchange code for GitHub access token for AppService CLI",
+        "description": "Description for Exchange code for GitHub access token for AppService CLI",
+        "operationId": "GenerateGithubAccessTokenForAppserviceCLIAsync",
+        "parameters": [
+          {
+            "name": "request",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AppserviceGithubTokenRequest"
+            },
+            "x-ms-client-flatten": true
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/AppserviceGithubToken"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "../2020-06-01/CommonDefinitions.json#/definitions/DefaultErrorResponse"
```

Comment:
Can you please add CommonDefinitions.json for 2020-09-01?

Reply:
Is it ok to keep using the 2020-06-01 CommonDefinitions? Since there are no changes to it, and we don't want to increment anything we don't need to

Reply:
I'm not sure if there is official guideline for this, but I think all models representing an api-version should be maintained under the api-version. It's easier to maintain (i.e. don't need to cross-check multiple api-versions when updating the model).

Reply:
Added